packages/ui-color-picker/src/ColorPreset/props.ts

```diff
@@ -67,10 +63,14 @@ type ColorPresetOwnProps = {
      * screenReaderLabel for the add new preset button
      */
     addNewPresetButtonScreenReaderLabel: string
-    maxHeight: string
+    selectColorLabel: string
+    removeColorLabel: string
     onPresetChange: (colors: ColorPresetOwnProps['colors']) => void
+    popoverAddButtonLabel: string
+    popoverCloseButtonLabel: string
```

Comment:
These are breaking too, some texts were hard-coded, and now became required props